packages/elemental/elemental/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,25 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,14 +1,18 @@
+ annotations:
++  catalog.cattle.io/auto-install: elemental-crd=match
+   catalog.cattle.io/certified: rancher
+   catalog.cattle.io/display-name: Elemental
+-  catalog.cattle.io/experimental: "true"
+   catalog.cattle.io/namespace: cattle-elemental-system
+   catalog.cattle.io/os: linux
+   catalog.cattle.io/provides-gvr: elemental.cattle.io/v1beta1
```

Comment:
You need to add the annotation `kube-version`  like this https://github.com/rancher/charts/blob/dev-v2.7/charts/epinio/102.0.3%2Bup1.8.1/Chart.yaml#L7

Reply:
You need to add `permist-os` annotation like this https://github.com/rancher/charts/blob/dev-v2.7/charts/epinio/102.0.3%2Bup1.8.1/Chart.yaml#L9

Reply:
Thanks, done!